src/styles.scss

```diff
@@ -1096,10 +1093,6 @@ div.fixed-top {
   background-color: white !important;
 }
 
-.dropdown-toggle::after {
-  display: none !important;
```

Comment:
https://github.com/congarevenuecloud/partner-commerce/pull/14.. This PR in partner commerce template you have this change.Can u=you reverify if we need this in both template or not?

Reply:
The same class was added thrice here so 2 of them were deleted.